module3/lessons/Week1/CSSFundamentals.md

```diff
@@ -0,0 +1,156 @@
+- `id` / `class` attributes on HTML elements that allow us to identify HTML elements in our CSS
+- `property` (CSS property) The name of a style property of an HTML element (e.g., color, border)
+- `value` The value that is paired with a given style property
+- `declaration` A property/value pair within a CSS rule
+- `declaration blocks` multiple declarations
+- `selector` used to target HTML elements on our web pages that we want to style
+- `rule` selector(s) and a declaration block come together to create a rule
+
+
+## What is CSS?
+CSS can do SO MUCH more than just add color or make things pretty. CSS can be used to create artwork like [this project](https://www.cssartist.com/inprogress) by Kassandra Sanchez. CSS can also make complex processes more intuitive and accessible users, like [TurboTax](https://turbotax.intuit.com/personal-taxes/online/) and [airbnb](https://www.airbnb.com/). CSS is a powerful tool for frontend developers. Let's jump in to see how it works!  
+
+It’s a “style sheet language” that lets you style the HTML elements on your page. CSS works _with_ HTML, but isn't HTML. CSS controls the positioning, sizing, colors, and specific fonts on your page. There is a `class` and `id` attribute available to use on __every__ HTML element. In addition to the plain old element names themselves, these attributes allow you to create "targets" for both your CSS and JavaScript. They are hooks that you can use to manipulate the look and behavior of your HTML elements.
+
+## Anatomy of a Basic CSS Rule
+
+![CSS Rule](/assets/images/module3/week1/css-rule.png)
+
+We can target an HTML element in CSS many ways:
+```css
+// by element name
+h1 {
+  color: red;
+}
+
+// by class name
+.primary-font {
+  font-family: "Montserrat", sans-serif;
+  font-size: 45px;
+}
+
+// by id name
+#about-page {
+  background-color: lightgrey;
+}
+
+// by a combination of above
+.primary-font p {
+  font-size: 15px;
+  color: darkgrey;
+}
+```
+
+<p class="codepen" data-height="300" data-theme-id="37918" data-default-tab="html,result" data-user="kaylaewood" data-slug-hash="mdEexwR" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;" data-pen-title="CSS Rules">
+  <span>See the Pen <a href="https://codepen.io/kaylaewood/pen/mdEexwR">
+  CSS Rules</a> by Kayla Wood (<a href="https://codepen.io/kaylaewood">@kaylaewood</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://static.codepen.io/assets/embed/ei.js"></script>
+
+<section class="call-to-action" markdown="1">
+### Explore
+
+With your partner, use the CodePen above to explore.
+* Uncomment each line in the CSS file at a time (starting at the top). Before you uncomment, make a prediction about what you will see on the page.
+* Do you notice a pattern for what styling is taking precedent?
+* Write down any questions that come up!
+</section>
```

Comment:
I like that students are exploring to figure out what takes precedent! I think might be helpful to also have the cascade order id -> class -> element written down somewhere. Maybe in a dropdown for less spoilers?